### PR TITLE
Remove tests

### PR DESCRIPTION
**Pull request by scsven**
Signed-off-by: Xiangyu Wang <xiangyu.wang@zilliz.com>

### CODE REVIEW

File: milvus/client/pool.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 from . import __version__
 from .grpc_handler import GrpcHandler, RegistryHandler
-from .http_handler import HttpHandler
```
Comment (XuanYang-cn) [inline milvus/client/pool.py:12]: Are we not going to support Http request?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+unittest:
+	python -m pytest tests
+
+package:
+	python -m build --sdist --wheel --outdir dist/.
```
Comment (XuanYang-cn) [inline Makefile:6]: Maybe better if we use `python3`

We need to add `build` module into requirements.txt

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: milvus/client/pool.py (hunk 1 above)
Comment (scsven) [reply milvus/client/pool.py:12]: These codes are outdated. And we will not support HTTP in near future.

File: Makefile (hunk 2 above)
Comment (scsven) [reply Makefile]: Changed to python3.
build already in requirements.txt.

**Pull request merged**